src/main/java/org/gradle/profiler/GradleBuildInvoker.java

```diff
@@ -10,6 +10,7 @@ public String toString() {
         }
     };
     static final GradleBuildInvoker AndroidStudio = new GradleBuildInvoker(GradleClientSpec.AndroidStudio, GradleDaemonReuse.WarmDaemonOnly);
+    static final GradleBuildInvoker AndroidStudioFirstSync = new GradleBuildInvoker(GradleClientSpec.AndroidStudioFirstSync, GradleDaemonReuse.ColdDaemonOnly);
```

Comment:
Android Studio does not re-use daemon after restart. So it always start with cold deamon in that case: I noticed that because we have checks in the gradle-profiler, that in case  `WarmDaemonOnly` is set but that is not the case exception is thrown.